### PR TITLE
Small change to db npm scripts

### PR DESCRIPTION
**Pull request by EliasJorgensen**
I noticed that when stopping the db, and using db:start to start it again, that it wouldn't start it, since that a container with name `expertsdb` already existed. 
For that reason, we should rename db:start to db:create, as it essentially works as a creation script, and not a starting script. We should also have a dedicated db:start script, that starts the `expertsdb` container again, if it has been stopped.

### CODE REVIEW

**Comment by yamalight** [comment]
@EliasJorgensen can you please also update README.md in `./server` to reflect that?

File: server/README.md [hunk 1]
```diff
@@ -33,7 +33,7 @@ To run the server locally, do the following:
 1. Clone this repository
 2. Enter `./server` folder
 3. Install dependencies with `npm install`
-4. Make sure you have local RethinkDB running (or start one using `npm run db:start`, requires docker)
+4. Make sure you have local RethinkDB running (or start one using `npm run db:create`, requires docker)
```
Comment (EliasJorgensen) [inline server/README.md]: I've updated the `npm run db:start` to `npm run db:create`

**Comment by yamalight** [comment]
Looks good, thanks!

File: server/README.md [hunk 2]
```diff
@@ -42,6 +42,7 @@ To run the server locally, do the following:
 Development requires you having Node.js v4 or later installed.  
 To run server for development just follow instuctions from [Using Node](#using-node) section.  
 To run test suite simply execute `npm test`.
+To see all available commands, see package.json.
```
Comment (EliasJorgensen) [inline server/README.md]: I also added a line, pointing people towards `package.json` to see all available commands.

**Pull request merged**